### DISCUSSION TITLE
What does the "action" property represent in Elastic?

### DISCUSSION THREAD

**Discussion by antmar904** [LETHAL-FORENSICS/MemProcFS-Analyzer#5] [General]
Hi.
When looking at events in Elastic particularly NFTS events, what do the action properties (RD, MOD, CRE, etc) mean?

**Comment by evild3ad** [comment]
You mean the ACTION column of "FS_Forensic_Timeline"?
https://github.com/ufrisk/MemProcFS/wiki/FS_Forensic_Timeline

I will ask Ulf if he may update this info in his wiki. Cheers!

RD     = Read
MOD = Modification Time
CRE   = Creation Time

**Reply by antmar904** [reply]
Thank you
